Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "ForwardDiff"
 uuid = "f6369f11-7733-5829-9624-2563aa707210"
-version = "0.10.25"
+version = "0.10.26"
```

Comment:
I believe @KristofferC prefers to let a few changes to build up before cutting a release?

Reply:
There's already an open PR for making a new release (with the NaNMath compat PR) which nobody has reviewed for weeks...